roles/vdm/tasks/storage.yaml

```diff
@@ -5,7 +5,7 @@
     cadence_number: "{{ V4_CFG_CADENCE_VERSION }}"
     existing: "{{ vdm_overlays }}"
     add:
-      - { transformers: "sas-storageclass.yaml", vdm: true, priority: 99 }
+      - { transformers: "sas-storageclass.yaml", vdm: true, priority: 49 }
```

Comment:
Yes, the intent was to place it as late in the `pre` items as possible, but in the `pre` section instead of `post` so that a transformer placed under site-config that in appears in the `user` section will have the opportunity to affect it.